<launch>

    <node name="radius" pkg="week2" type="radius.py" />
    <node name="compute_ang_vel" pkg="week2" type="ang_vel_server.py" />
    <node name="Turtle_Path" pkg="week2" type="ang_vel_sub.py" output="screen" />

</launch>